### PR TITLE
employee

### PR DESCRIPTION
**Pull request by mmayen**
created employee object

### CODE REVIEW

File: src/client/src/models/Employee.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default class Employee {
+    constructor(id, name) {
+    this.id=id;
+    this.name=name;
+    }
+
+    Edit(id, name) {
```
Comment (merekorlowski) [inline src/client/src/models/Employee.js]: We're going to follow the convention of camalCase for variable and function names. Change "Edit" to "edit".

File: src/client/src/models/Employee.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default class Employee {
+    constructor(id, name) {
+    this.id=id;
```
Comment (merekorlowski) [inline src/client/src/models/Employee.js]: Make sure to indent these using tab.

**Comment by merekorlowski** [comment]
Looks good.

File: src/client/src/models/Employee.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 export default class Employee {
-    constructor(id, name) {
-    this.id=id;
-    this.name=name;
-    }
+    
+        constructor(id, name) {
```
Comment (karim2506) [inline src/client/src/models/Employee.js]: Shift the whole constructor to the left by 1 indent

File: src/client/src/models/Employee.js [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
+            this.id=id;
+            this.name=name;
+        }
 
-    Edit(id, name) {
-        this.id = id;
-        this.name = name;
-    }
+        edit(id, name) {
```
Comment (karim2506) [inline src/client/src/models/Employee.js]: Shift the edit function to the left by 1 indent

**Pull request merged**